src/lib389/lib389/utils.py

```diff
@@ -1242,9 +1306,9 @@ def ds_is_related(relation, *ver, instance=None):
     if len(ver) > 1:
         for cmp_ver in ver:
             if cmp_ver.startswith(ds_ver[:3]):
-                return ops[relation](LegacyVersion(ds_ver),LegacyVersion(cmp_ver))
+                return ops[relation](DSVersion(ds_ver),DSVersion(cmp_ver))
```

Comment:
I think pep8 will want a space after the comma

Reply:
Done